src/dmd/attrib.d

```diff
@@ -929,6 +928,7 @@ extern (C++) final class StaticIfDeclaration : ConditionalDeclaration
 {
     ScopeDsymbol scopesym;
     bool addisdone;
+    bool onStack = false;
```

Comment:
All new public symbols should have documentation.

Reply:
It does not need to be public, but neither does addisdone. Made both private and added comments.